src/main/resources/db/changelog/changesets/changelog_2021-12-24T14:47:39Z.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.10.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.10.xsd">
+    <changeSet author="amarsli (generated)" id="1640357270577-5">
+        <createTable tableName="branch_status_modification">
+            <column name="action" type="VARCHAR(255)"/>
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="branch_status_modificationPK"/>
+            </column>
+        </createTable>
+    </changeSet>
```

Comment:
Don't know why liquibase generates these subsequent changesets on two windings transformer creation foreign keys ...
Maybe, you can remove them ?

Reply:
?

Reply:
These additiional changesets were present in the first version of the file ...
But they disapeared in the second version, so it's ok ...